### PR TITLE
Fix: Remove the nonsense button that appears in the nav map

### PR DESCRIPTION
**Pull request by 0xD-Fabio**
## What does this PR change?

Remove the nonsense button that appears in the nav map

## How to test the changes?

Open the map, and check if the button is not there.
image

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/Resources/NavMap.prefab [hunk 1]
```diff
@@ -1078,7 +1078,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 1
+  m_IsActive: 0
```
Comment (dalkia) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/Resources/NavMap.prefab:1081]: I would delete the button alltogether.

**Review by dalkia — CHANGES_REQUESTED** [review]
Looks good! Leaving just a small suggestion

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/Resources/NavMap.prefab (hunk 1 above)
Comment (0xD-Fabio) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/Resources/NavMap.prefab:1081]: My bad, I forgot to push the deleted button prefab. Doing it now ;)

**Review by dalkia — APPROVED** [review]

**Review by kwssbocian — APPROVED** [review]
Fix has been implemented correctly.
✔️The button located in the top right corner of the nav map has been successfully removed.

**Pull request merged**